presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -145,6 +145,7 @@
     public static final String ITERATIVE_OPTIMIZER = "iterative_optimizer_enabled";
     public static final String ITERATIVE_OPTIMIZER_TIMEOUT = "iterative_optimizer_timeout";
     public static final String QUERY_ANALYZER_TIMEOUT = "query_analyzer_timeout";
+    public static final String SEMANTIC_ANALYZER_TIMEOUT = "semantic_analyzer_timeout";
```

Comment:
what's the difference between the query analyzer and the semantic analyzer?

Reply:
never mind, i saw some explanation in the PR description.  This is to cover explain queries.  Can we use the same timeout instead of adding a new configuration?

Reply:
Done, use the same timeout configuration now.